app/assets/javascripts/views/notes/notes_view.ts

```diff
@@ -365,7 +365,7 @@ class NotesViewCtrl extends PureViewCtrl<unknown, NotesCtrlState> {
     await this.reloadNotes();
     await this.appState.noteTags.reloadTags();
     const noteTitleEditorElement = document.getElementById('note-title-editor');
-    noteTitleEditorElement?.focus();
+    if(focusNewNote) noteTitleEditorElement?.focus();
```

Comment:
```suggestion
    if (focusNewNote) {
      noteTitleEditorElement?.focus();
    }
```

Best to avoid inline conditionals for better readability 🙂